prow/config.yaml

```diff
@@ -200,29 +200,19 @@ presubmits:
         - "--upload=gs://kubernetes-jenkins/pr-logs"
         - "--clean"
         env:
-        - name: DOCKER_IN_DOCKER_ENABLED
-          value: true
         - name: GOOGLE_APPLICATION_CREDENTIALS
           value: /etc/service-account/service-account.json
-        # docker-in-docker needs privileged mode
-        securityContext:
-          privileged: true
         volumeMounts:
         - name: service
           mountPath: /etc/service-account
           readOnly: true
-        - name: docker-graph
-          mountPath: /docker-graph
         ports:
         - containerPort: 9999
           hostPort: 9999
```

Comment:
you can remove the ports too :-)

Reply:
Sorry I squashed the commits so I can't see the file. Which jobs should I remove the ports from?

Reply:
the ones don't need docker